api/src/main/scala/pl/touk/nussknacker/engine/api/process/ExpressionConfig.scala

```diff
@@ -31,7 +31,9 @@ object ExpressionConfig {
   val empty = ExpressionConfig(Map.empty, Nil)
 
   // Those types must be explicitly provided because can be used in dynamic parameters
-  val standardEditorClasses: List[Class[_]] = List(classOf[LocalDate], classOf[LocalTime], classOf[LocalDateTime], classOf[Duration], classOf[Period])
+  val standardEditorClasses: List[Class[_]] = List(
+    classOf[LocalDate], classOf[LocalTime], classOf[LocalDateTime], classOf[Duration], classOf[Period], classOf[UUID]
```

Comment:
`UUID` is not and "editor" class. By "editor" I mean parameter editor. Let's add a new group like `standardClasses` and sum them up

Reply:
:+1: